### DISCUSSION TITLE
How to sync hudi table with delta and iceberg while writing using spark.

### DISCUSSION THREAD

**Discussion by lalit2001** [apache/incubator-xtable#574] [Q&A] [answered]
How can I sync metadata with iceberg and delta while writing hudi table using py spark? is there any way to do it or this functionality is not present .

**Comment by vinishjail97** [comment, marked as answer]
Yes you can.
https://github.com/apache/incubator-xtable/blob/main/xtable-hudi-support/xtable-hudi-support-extensions/README.md
